### ISSUE TITLE
Wrong display in wallabag

### ISSUE THREAD

**Issue by j0k3r** [wallabag/wallabag#3940]
*Sent by Fiona Halar and automatically created by email*

---

https://www.gizmodo.com.au/2019/04/a-common-preservative-in-cheese-and-bread-could-negatively-affect-our-metabolism-study-finds/

**Comment by techexo** [comment]
I encountered the same issue today with [this article](https://www.gizmodo.com.au/2019/01/i-tried-to-block-amazon-from-my-life-it-was-impossible/).

Looking at the [site configuration](https://github.com/fivefilters/ftr-site-config/blob/master/gizmodo.com.au.txt), there is the following lines:
```
find_string: <meta http-equiv="refresh"
replace_string: <meta http-equiv="disabled-refresh"
```
relative to this code
```html
<noscript>
    <meta http-equiv="refresh" content="0;url=/?nojs=1">
</noscript>
```

Trying to fetch these articles fail with `Meta refresh redirect found (http-equiv="refresh"), new URL: https://www.gizmodo.com.au/?nojs=1`, which enters an endless loop on the website.

I tried loading my article in Firefox with JS disabled, and the same redirection is applied. It would seem that the hack in the site configuration file doesn't work anymore.

Can someone see a way to avoid this issue?
